sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -1,17 +1,11 @@
 # Release History
 
-## 12.3.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 12.2.4 (2022-06-07)
```

Comment:
If no package that depends on common is GA'ing yet, is it necessary to GA this right now? Given blobs is shipping beta, maybe this should be as well.

Is the bug fix critical for customers on older GA releases of the SDK?

Reply:
That's true, but it seems strange to rev and ship a new version on a shared package which no end customer using GA'd storage service packages can benefit from. That said, it's good that we are only, and correctly, only incrementing the patch version.

Do we even have to ship storage common at all this month, or can we skip it?

Reply:
The new blob release 12.5.0 beta1 depends on it. So we have to ship a common package, either GA or beta.

Reply:
Alright, it's your call :)